hindi/bollywood.md

```diff
@@ -2,8 +2,7 @@
 
 बॉलीवुड में आपका स्वागत है !
 
-आपका पसंदीदा अभिनेता कौन है?      
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.